bin/upload_new_release_to_s3.py

```diff
@@ -0,0 +1,47 @@
+import boto3
+import os
+import datetime
+import argparse
+
+parser = argparse.ArgumentParser(description='Usage')
+parser.add_argument('-r','--release-name', type=str,
+                    help="Name of release")
+args = parser.parse_args()
+
+if not args.release_name:
+  raise Exception("Please include a name for your release")
+
+s3 = boto3.client('s3')
+now = datetime.date.today()
+
+def recursive_upload(dir = None, parent_dirs=None):
+  home_path = os.path.join(os.path.expanduser('~'), "aitt_release_data")
+  if parent_dirs and dir:
+    full_path = os.path.join(home_path, parent_dirs, dir)
+    new_parent_dirs = os.path.join(parent_dirs, dir)
+  elif dir and not parent_dirs:
+    full_path = os.path.join(home_path, dir)
+    new_parent_dirs = dir
+  else:
+    full_path = home_path
+    new_parent_dirs = None
+  
+  files = os.listdir(full_path)
+  if '.git' in files:
+    files.remove('.git')
+    files.remove('README.md')
```

Comment:
does `README.md` always exist when `.git` does? this would technically crash if not.

You could import:

```
from contextlib import suppress
```

And then:

```
files = os.listdir(full_path)
with suppress(ValueError):
    files.remove('.git')
    files.remove('README.md')
for f in files:
...
```

To just try to remove it and ignore the errors when it isn't found.

Reply:
This isn't removing them from the repo, it's just removing them from the file list that's being parsed for the upload to S3. We don't need to upload the Readme and .git folder.

Reply:
This should only return true at the root of the aitt_release_data repo, and it's a temp solution while we get cicd setup. I can add the suppress error code, but I don't think it's necessary, since this script has such a specific stopgap use.

Reply:
I think this all exposes the issue that the config we plan to upload should really be isolated to a single directory so that we can recursively copy it without special additional logic. to consider doing in the future